### ISSUE TITLE
Mobs Do Not Spawn in Server

### ISSUE THREAD

**Issue by Jissee** [Tslat/Advent-Of-Ascension#2909]
Hi, I really like your mod. But when I load the mod in my server to play with my friends, I noticed that there are not any mobs from your mod in the world. However, all the mobs in Minecraft can spawn and mobs can spawn correctly in singleplayer mode. How can I deal with the problem? I am using AoA3-3.5.4-1.16.5.jar and the server is arclight-forge-1.16.5-1.0.21-SNAPSHOT.jar. Here are the mods and plugins I used in my server.
![plugins](https://user-images.githubusercontent.com/50765103/144780208-2ae860f8-1109-4b05-b231-d90d6e027f5b.PNG)
![mods](https://user-images.githubusercontent.com/50765103/144780213-9f4e7cb7-612c-476b-b729-b2b0a1cec936.PNG)
.

**Comment by EatingSilencer** [comment]
I don't see anything that looks like it would do it right off the bat. You'll probably have to take out half the mods, see if it fixes, and keep halving until you find the culprit mod.

**Comment by Tslat** [comment]
That doesn't look like your server's mod list

You have several client-only mods in that list.

**Comment by Jissee** [comment]
Hello! I have finally figured out and solved the problem. I am running a server(arclight) that injects bukkit plugins into the forge server. The world with the problem was generated and loaded by the plugin "Multiverse". However, all the plugins are intended for vanilla versions but not for modded versions and, maybe, mod data cannot be recognized by the plugins. As a result, the mod mobs disappeared. To deal with the problem, I tried to load the world directly into the server and made it the root world. Now everything can run smoothly. Sorry to trouble you and I hope that the mod can be compatible with Minecraft version 1.18.1 soon.
